package.json

```diff
@@ -31,13 +31,15 @@
     "@emotion/react": "^11.1.4",
```

Comment:
did we want to use npm or yarn for this? i'm cool with either

Reply:
I was sticking with yarn, so we can remove the package-lock